ui/widget.js

```diff
@@ -12,6 +12,7 @@
 //>>description: Provides a factory for creating stateful widgets with a common API.
 //>>docs: http://api.jqueryui.com/jQuery.widget/
 //>>demos: http://jqueryui.com/widget/
+//>>css.theme: ../themes/base/theme.css
```

Comment:
Unlike `core.js`, there's actual usage for CSS classes in here, but then by itself, `$.Widget` doesn't really do anything. You can build a widget that never touches the DOM, so I don't think this reference should be here. Again, input from @arschmitz would be very welcome.